react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {html: false, css: false, js: false};
+    this.jsOn = this.jsOn.bind(this);
+    this.jsRender = this.jsRender.bind(this);
+    this.cssOn = this.cssOn.bind(this);
+    this.htmlOn = this.htmlOn.bind(this);
+    this.htmlRender = this.htmlRender.bind(this);
+    this.cssRender = this.cssRender.bind(this);
+    this.start = this.start.bind(this);
+    this.startRender = this.startRender.bind(this);
+  }
+  jsOn() {
+    this.setState({ html: false, css: false, js: true});
+  }
+  cssOn(){
+    this.setState({html: false, css: true, js: false})
+  }
+  htmlOn(){
+    this.setState({html: true, css: false, js: false})
+  }
+  start() {
+    this.setState({ html: false, css: false, js: false })
+  }
```

Comment:
This is way too specific, you should only have one click handler function. You want your accordion to work with any data passed in that is formatted correctly.

A better approach to the accordion is to store only the `id` of the open item in state.